app/services/fees/create_pay_in_advance_service.rb

```diff
@@ -126,5 +131,17 @@ def deliver_webhooks
 
       result.fees.each { |f| SendWebhookJob.perform_later('fee.created', f) }
     end
+
+    def update_event_metadata(aggregation_result:)
+      unless aggregation_result.current_aggregation.nil?
```

Comment:
I see a potential issue with attaching these details to the event:

The `current_aggregation` and `max_aggregation` properties as relative to the subscription (via the boundaries). 
Since the same billable metric could be used on different plans having different duration and so depend to multiple subscriptions, the values stored at event level could we wrong.
I think we should store these values on an entity attached to the subscription and not on the event itself.